pyuvdata/uvdata/uvdata.py

```diff
@@ -1595,13 +1694,12 @@ def merge_phase_centers(self, catname1, catname2, force_merge=False):
 
         Parameters
         ----------
-        catname1 : str
-            String containing the name of the first phase center. Note that this name
-            will be preserved in the UVData object.
-        catname2 : str
-            String containing the name of the second phase center, which will be merged
-            into the first phase center. Note that once the merge is complete, all
-            information about this phase center is removed.
+        cat_name : str
+            String containing the name of the phase center(s) to merge. Either cat_name
+            or cat_id must be specified.
```

Comment:
Looks like this can be multiple phase centers, can this be a list?

Reply:
Added handling for lists of str in e1cbd00b.